tests/test_icecream.py

```diff
@@ -600,7 +588,7 @@ def testMultilineContainerArgs(self):
                     list(range(15))])
 
         lines = err.getvalue().strip().splitlines()
-        self.assertRegexpMatches(
+        self.assertRegex(
             lines[0],
```

Comment:
Here ?

Reply:
assertRegexpMatches is been replaced by assertRegex. So this is future proof